scripts/pyproject.toml

```diff
@@ -9,21 +9,21 @@ authors = [
 ]
 description = "data prep scripts"
 # readme = ""
-requires-python = ">=3.10"
+requires-python = ">=3.8, <3.9"
```

Comment:
probably this is a good place to have a comment that the nuc-morph-analysis dependency is only for convert_nucmorph_data and that's what's keeping this python version at 3.8